modules/reference/pages/cli-container.adoc

```diff
@@ -0,0 +1,39 @@
+[[ref-cli-uyuni-tools]]
+= Uyuni Tools
+
+
+[[uyuni-tools-description]]
+
+== Description
+The [package]``uyuni-tools`` package has components as subpackages with CLI tools:
+
+[[uyuni-tools-components]]
+.Uyuni Tools CLI Tool Components
+[cols="1,1,1,1", options="header"]
+|===
+| Package name            | Tool name               | Installed on | Use for
+| ``uyuni-tools``         | ``uyunictl``            | Host         | day-to-day operations against a containerized {productname} server
```

Comment:
Keep in mind that the package doesn't exist yet. We'll need to make sure this is the final name